packages/headless/docs/ControllersParser.ts

```diff
@@ -0,0 +1,59 @@
+import {Config, Controller, DocGen, Func, Source} from './DocJsonTypes';
```

Comment:
It would be nice to have filenames in kebab-case to be consistent with the rest of the headless project. e.g.
`controller-parser.ts`